sentry-samples/sentry-samples-spring/src/main/resources/sentry.properties

```diff
@@ -1 +1,3 @@
 debug=true
+# it didn't work, looks like every frame was inApp=false
+in-app-includes="io.sentry.samples"
```

Comment:
@maciejwalkowiak does spring (non boot) reads from external resources?

Reply:
It does but there is a bug with merging options, I'll fix it in a minute.